admin/broadleaf-admin-module/src/main/java/org/broadleafcommerce/admin/persistence/validation/OfferQualifyingCriteriaValidator.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * #%L
+ * BroadleafCommerce Admin Module
+ * %%
+ * Copyright (C) 2009 - 2016 Broadleaf Commerce
```

Comment:
The end year should have been current. Did you copy header instead of generating? It should generate headers when u run mvn install. Anyway change it to 2022.